### PR TITLE
Add new quest for parking access

### LINKED ISSUE #100: Quest: Is this parking area Public, Private, Customers or Other?
Apps that use this data:
 * http://product.itoworld.com/map/7
 * https://play.google.com/store/apps/details?id=fr.lcdm.freeparking&hl=en

Its very easy to tag an area as amenity=parking, but less easy to determine what kind of parking it is.

### PR DESCRIPTION
**Pull request by ENT8R**
This PR adds a new quest to determine the accessibility of a parking. It is a simple radio button group with three options. This PR fixes #100.
It looks like this:

I am pretty unhappy with the wording of the first option... Does anybody have another idea?

### CODE REVIEW

**Comment by westnordost** [comment]
- It's private
- It's public
- Only for customers

?

**Comment by westnordost** [comment]
And perhaps the question could be rephrased so that an answer option like "people who pay for it" doesn't lend itself as a probable answer.

**Comment by westnordost** [comment]
I'm going to pull it in now so that it can go into v4 still.

**Comment by ENT8R** [comment]
Great! Thank you!

File: app/src/main/java/de/westnordost/streetcomplete/quests/parking_access/AddParkingAccess.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddParkingAccess extends SimpleOverpassQuestType
+{
+	@Inject public AddParkingAccess(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override
+	protected String getTagFilters() {
+		return "nodes, ways with amenity=parking and (!access or access=unknown)";
```
Comment (rugk) [inline app/src/main/java/de/westnordost/streetcomplete/quests/parking_access/AddParkingAccess.java]: BTW why do you use different queries here and [here](https://github.com/westnordost/StreetComplete/pull/825/files#diff-21e98c586106a4fcb30e234c8c81c5a9R25)? Also, the parking type has [yet another query](https://github.com/westnordost/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/parking_type/AddParkingType.java#L24)…
Would not it be better to unify this? Maybe even in a base class?

**Pull request merged**